### DISCUSSION TITLE
Windhawk Windows 11 Start Menu Syler: TranslucentStartMenu

### DISCUSSION THREAD

**Discussion by donzzy** [ramensoftware/windhawk-mods#966] [Q&A]
![white back](https://github.com/user-attachments/assets/43c82ae9-ab4d-41b6-bbb9-17c2bbdf6528)
I love this particular mod. However this is the only problem I have whenever the background is plain white. Is there a way  to automatically change the icon text label from white to gray or black?

**Comment by m417z** [comment]
It was discussed here: https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues/76.

**Reply by donzzy** [reply]
forgive i'm such a noob. where do I put the added code?

**Reply by m417z** [reply]
It's best to ask in that thread.
